brave-kafka/pom.xml

```diff
@@ -57,7 +57,7 @@
       <artifactId>brave</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.zipkin.java</groupId>
+      <groupId>io.zipkin.zipkin2</groupId>
```

Comment:
I don't think you'll need this dep except maybe in testing as we don't access endpoint directly anymore